src/compute/src/sink/persist_sink.rs

```diff
@@ -1181,3 +1191,87 @@ where
     let token = Rc::new(shutdown_button.press_on_drop());
     (output_stream, token)
 }
+
+/// This is for REFRESH options on materialized views. It adds an operator that rounds up the
+/// timestamps of data and frontiers to the time of the next refresh. See
+/// `doc/developer/design/20231027_refresh_every_mv.md`.
+///
+/// Note that this currently only works with 1-dim timestamps. (This is not an issue for WMR,
+/// because iteration numbers should disappear by the time the data gets to the Persist sink.)
+fn round_up<G>(
```

Comment:
Can we rename the operator (both the function name here and the name we pass to `OperatorBuilder::new` below) to something that makes it more obvious what it's doing? I propose `apply_refresh`.

Reply:
Done